Scenes/Player.tscn

```diff
@@ -209,7 +209,7 @@ animations = [{
 }],
 "loop": true,
 "name": &"walk",
-"speed": 5.0
+"speed": 25.0
```

Comment:
Why 25 instead of 30 or 60?

Reply:
For this sprite set, 25 is a good number for a fluid motion. Any higher, and it looked like it was sprinting to go do a line of coke of a stripper's bum.

Reply:
Fair enough